introduction.md

```diff
@@ -0,0 +1 @@
+Hi, this is an introduction. I am FileNotFound and i am a computer science student in germany. Currently I am writing my master what includes using GitHub, so i take the corresponding Codeschool course.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.